### PR TITLE
chore: remove allowHaveYourSay and allowGetUpdates feature flags from the codebase

### PR DESCRIPTION
**Pull request by cprieststephens**
## Describe your changes

2406: Feature Flag removal - FEATURE_ALLOW_HAVE_YOUR _SAY/ FEATURE_ALLOW_GET_UPDATES
https://pins-ds.atlassian.net/jira/software/c/projects/ASB/boards/166?selectedIssue=ASB-2406

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [x] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: packages/forms-web-app/src/pages/projects/_middleware/middleware.component.test.js [hunk 1]
```diff
@@ -17,11 +17,7 @@ jest.mock('../../../config', () => {
 			...originalConfig.featureFlag,
 			allowProjectInformation: false,
 			hideProjectTimelineLink: false,
-			allowDocumentLibrary: false,
-			allowExaminationTimetable: false,
-			hideAllExaminationDocumentsLink: false,
-			allowHaveYourSay: false,
-			allowGetUpdates: false
```
[the remark this answers is not in the document]
Comment (cprieststephens) [reply packages/forms-web-app/src/pages/projects/_middleware/middleware.component.test.js:24]: Thanks @christopher-semmens  - FEATURE_ALLOW_DOCUMENT_LIBRARY and FEATURE_ALLOW_EXAMINATION_TIMETABLE were removed in a separate PR. I spotted they were still being mocked here so removed them here too. Happy to change it though.

[the remark this answers is not in the document]
Comment (christopher-semmens) [reply packages/forms-web-app/src/pages/projects/_middleware/middleware.component.test.js:24]: @cprieststephens Thank you, I think that it is fine to leave them here

**Review by christopher-semmens — APPROVED** [review]

File: packages/forms-web-app/src/pages/projects/_middleware/middleware.component.test.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (cprieststephens) [reply packages/forms-web-app/src/pages/projects/_middleware/middleware.component.test.js:24]: @christopher-semmens So the two additional mocked feature flags need be added back in to this test? Just checking I understand correctly before I make the change.

**Pull request merged**